mrp_hook/models/mrp_bom.py

```diff
@@ -0,0 +1,150 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Serv. Tecnol. Avanzados - Pedro M. Baeza
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, api, tools, _
+from openerp.exceptions import Warning as UserError
+from openerp.addons.product import _common
+
+
+class MrpBom(models.Model):
+    _inherit = 'mrp.bom'
+
+    @api.model
+    def _factor(self, factor, product_efficiency, product_rounding):
```

Comment:
Can you document this a little better, I understand that you can force a factor, but "why"?

factor / NUMBER NEAR TO 0 == INFINITE , why the denominator is 1.0 should not be forced the factor = 0?